# This patch file is from Qualcomm Technologies, Inc. and is provided under the following license:
#
# Copyright (c) 2025 Qualcomm Technologies, Inc. All rights reserved.
#
# SPDX-License-Identifier: BSD-3-Clause-Clear

commit e1430350808711cd5383194e5dc56680c28a46b7
Author: Ankurkumar Satasiya <asatasiy@qti.qualcomm.com>
Date:   Wed May 21 13:37:17 2025 +0200

    Blow milestone fuse.
    
    Earlier, we did not set the Param 0 to any value and hence, it was
    blowing TZ_HLOS_IMG_TAMPER_FUSE instead of TZ_HLOS_BL_MILESTONE_FUSE.

Signed-off-by: Ankurkumar Satasiya <asatasiy@qti.qualcomm.com>
diff --git a/QcomModulePkg/Library/avb/VerifiedBoot.c b/QcomModulePkg/Library/avb/VerifiedBoot.c
index a3c9bf6..b8a46ee 100644
--- a/QcomModulePkg/Library/avb/VerifiedBoot.c
+++ b/QcomModulePkg/Library/avb/VerifiedBoot.c
@@ -1325,6 +1325,7 @@ TzBlowSwFuse (VOID)
     return Status;
   }
 
+  Param[0] = TZ_HLOS_BL_MILESTONE_FUSE;
   Status = pQcomScmProtocol->ScmSipSysCall(pQcomScmProtocol, TZ_BLOW_SW_FUSE_ID,
       TZ_BLOW_SW_FUSE_ID_PARAM_ID, Param, Results);
   if (Status != EFI_SUCCESS) {
